src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -189,7 +189,7 @@ def test_ad_invalid_iteration_space():
     # LFRicArgDescriptor object
```

Comment:
Since you were so kind to remove the futurising statements in other touched modules, would you mind doing it here as well? 😃